### PR TITLE
Fixes #9770 - add functionality to create hosts from hostgroups

### PR DESCRIPTION
**Pull request by nofaralfasi**
Writing an integration test for this feature is currently in progress.

### CODE REVIEW

File: app/views/hostgroups/index.html.erb [hunk 1]
```diff
@@ -25,6 +25,7 @@
         </td>
         <td>
           <%= action_buttons(
+                      display_link_if_authorized((link_to _('Create Host'), new_host_path(hostgroup_id: hostgroup.id))),
```
Comment (Ron-Lavi) [inline app/views/hostgroups/index.html.erb]: let's move it under `Nest` for now (that used to be visible as the main action)
or create a post in the community about what should be the main action?

Comment (Ron-Lavi) [inline app/views/hostgroups/index.html.erb]: let's move it under `Nest` for now (that used to be visible as the main action)
or create a post in the community about what should be the main action?

**Review by Ron-Lavi — COMMENTED** [review]
Works well @nofaralfasi !
can you also rebase the commit?

**Comment by Ron-Lavi** [comment]
ok to test

**Review by Ron-Lavi — APPROVED** [review]
Thanks @nofaralfasi !

**Comment by ares** [comment]
Nice and simple :+1:

**Pull request merged**